src/app-layout/visual-refresh-toolbar/drawer/index.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export { createWidgetizedAppLayoutDrawer, AppLayoutDrawerImplementation } from './local-drawer';
+export { createWidgetizedAppLayoutGlobalDrawer, AppLayoutGlobalDrawerImplementation } from './global-drawer';
```

Comment:
NB: to complete this widgetization, we need to make some changes on the internal counterpart too. Let's keep a reminder of this somewhere